### ISSUE TITLE
Current Sprite onend?

### ISSUE THREAD

**Issue by emreaypar** [goldfire/howler.js#854]
Hi,
I'm using sprites to play audio files sequentially. And i'm using the sprites demo as a guide. I can't seem to find out how to trigger an "onend" event for the current playing sprite. i wrote a next() function to get the next sprite but i don't think it works.
The sprites demo uses this:
`self.sound.once('end', function() {
  self.next();
  // var index = self.sounds.indexOf(elm);
  // if (index >= 0) {
  // self.sounds.splice(index, 1);
  // window[key].removeChild(elm);
  // }
}, id);`

At the moment, i'm tracking the current sprite by doing something like this in the step function:
`if ((seek - start) >= self.sound.duration(self.currentSpriteID)){
  self.next();
};`

it works but i'm curious if it's the right way of doing this or am i missing something from the documentation?

Thanks,
Emre

**Comment by abithyzis** [comment]
`.play('sprite part')` returns an id which you can use when listening to the onend event.

```
var first = mysprite.play('first');
instructions.on('end', function(){mysprite.play('second')}, first);
```
so you probably need to add on onend listener for every sprite part you have.

**Comment by emreaypar** [comment]
I checked the sprites demo on the howlerjs.com web page and the sprites behave as intended. I think the demo uses v2.0.4. I downloaded v2.0.4 and 2.0.5 and the same sprites example works on v2.0.4 and doesn't work on v2.0.5
I guess something about sprites got broke from v2.0.4 to v2.0.5

**Comment by goldfire** [comment]
This should be fixed with 2.0.6, which I'll push out shortly. Let me know if you are still seeing the issue and I'll reopen.
